lbrynet/core/Wallet.py

```diff
@@ -738,20 +764,26 @@ def get_transaction(self, txid):
     def get_claim_metadata_for_sd_hash(self, sd_hash):
         return self._get_claim_metadata_for_sd_hash(sd_hash)
 
-    def get_balance(self):
-        return self.wallet_balance - self.total_reserved_points - sum(self.queued_payments.values())
+    def get_balance(self, address=None):
```

Comment:
Hey Jack, thanks for the feedback will take a look over it now and makes some changes.